amr-wind/utilities/sampling/SamplerBase.H

```diff
@@ -45,6 +45,9 @@ public:
     //! Populate the vector with coordinates of the sampling locations
     virtual void sampling_locations(SampleLocType&) const = 0;
 
+    //! Update the sampling locations
+    virtual void update_sampling_locations() {}
+
```

Comment:
```suggestion
```

Not needed based on the actual final implementation of `Sampling.cpp`

Reply:
The implementation in `Sampling.cpp` calls this function. Should I be doing this differently?